requirements.txt

```diff
@@ -21,6 +21,7 @@ scipy
 sh
 sphinx
 sphinx_rtd_theme
+torch>=1.1
```

Comment:
I'm not sure we want to add this here specifically, since there are many alternative ways of installing pytorch - this will use pip, while one might want to use conda and also might want to install with CUDA etc. @stephenroller do you have any thoughts on this?

Reply:
Yes, what Kurt said.

Reply:
Okay I can remove this from here, but certain Circle CI tests like os_x tests specifically install pytorch 1.0.0 and hence those tests will fail as they are doing now.

Also, I am not sure how to deal with the beam search failing on this PR, Any thoughts ?